planner/core/physical_plans.go

```diff
@@ -545,6 +549,22 @@ type PhysicalWindow struct {
 	Frame           *WindowFrame
 }
 
+// PhysicalPartition represents a partition plan.
+type PhysicalPartition struct {
+	physicalSchemaProducer
```

Comment:
`PhysicalPartition` does not change schema for now, so is it better to use `basePhysicalPlan`?